example/test_problem/ELBDM/UniformGranule/generate_make.sh

```diff
@@ -0,0 +1,6 @@
+# This script should run in the same directory as configure.py
+
+PYTHON=python3
+
+${PYTHON} configure.py --machine=eureka_intel --mpi=true --hdf5=true --fftw=FFTW3 --gpu=true \
+                       --model=ELBDM --gravity=true --passive=1
```

Comment:
Please merge the latest `psidm` branch, which supports a default `--machine`.
```suggestion
${PYTHON} configure.py --mpi=true --hdf5=true --fftw=FFTW3 --gpu=true \
                       --model=ELBDM --gravity=true --passive=1 "$@"
```